server/gitrest/packages/gitrest-base/src/utils/nodegitManager.ts

```diff
@@ -193,7 +193,11 @@ export class NodegitRepositoryManager implements IRepositoryManager {
             throw new NetworkError(400, "Invalid input");
         }
 
-        const signature = nodegit.Signature.create(commit.author.name, commit.author.email, Math.floor(date), 0);
+        const signature = nodegit.Signature.create(
+            commit.author.name,
+            commit.author.email,
+            Math.floor(date / 1000),
```

Comment:
nit: comment explaining? even just `Math.floor(date / 1000), /* timestamp in milliseconds */`